### DISCUSSION TITLE
Setting the root command to use the name of the executable

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#1180] [Q&A] [answered]
@ganigeorgiev Currently the root command executable is setup with a hardwired "pocketbase" for the name.  This appears in the help.

```golang
RootCmd: &cobra.Command{
	Use:     "pocketbase",
}
```

This could be made better for using Pocketbase as a framework.

```golang
var commandName string
if path, err := os.Executable(); err != nil {
	commandName = "pocketbase"
} else {
	commandName = filepath.Base(path)
}

RootCmd: &cobra.Command{
	Use:     commandName,
}
```

Otherwise, I'll just be monkey patching it into my root command setup by doing this.

```golang
if path, err := os.Executable(); err != nil {
	rootCmd.Use = "pocketbase"
} else {
	rootCmd.Use = filepath.Base(path)
}
```

**Comment by ganigeorgiev** [comment, marked as answer]
Yes, we can change it. I'll consider updating it in the next release.

**Comment by IwateKyle** [comment]
I second this suggestion.  I love pocketbase.  I have just started using it and found that after having created a database with pocketbase, I *must* use the exact same path convention every time starting up the database. For example, the following are not equivalent
-------------
 pocketbase serve
~/bin/pocketbase serve
../app/pocketbase serve
------------
It seems that the database creates a unique database for each command path.  This makes it difficult for development.   And the way I discovered this was that I started off with pocketbase, created a few collections with data, then found that I could benefit from using the framework option.  After recreating the framework, I also modified the name of the app to represent my app.  After starting the app, the database tried  to create a new database.

-------------------
- started with : pocketbase serve
- changed to : myapp serve
- original data not reachable when using myapp
------------------

**Reply by ganigeorgiev** [reply]
@IwateKyle I don't think this discussion is related to your issue and it is already resolved. I've answered to your question in your other discussion https://github.com/pocketbase/pocketbase/discussions/2143.
